### ISSUE TITLE
Launcher Wont Identify the mod

### ISSUE THREAD

**Issue by Spirit117** [long-war-2/lwotc#377]
Hi guys, not sure if this is the right place for this, but I could use some help. Ive unzipped the mod files into my xcom2 game directory E:\Steam\steamapps\common\XCOM 2/Xcom2-WarOfTheChosen/Mods (xcom 2 runs on my second drive). I see the folders for both LWOTC and Community highlander in that folder, but whenever I launch my game, my mod launcher doesnt display the mods as launchable. Have tried uninstalling and reinstalling Xcom2, as well as removing any downloaded mods from steam workshop. Any help is appreciated.

**Comment by beugsman** [comment]
I also have this problem :(

**Comment by Maebalzurakin** [comment]
Under settings in the launcher, you have to create a path to that mod directory.

**Comment by beugsman** [comment]
> Under settings in the launcher, you have to create a path to that mod directory. You probably only have one path listed, which is your steam directory. Add a path to where you put the mods. Then close and relaunch and the highlander and long war should be there.

Thankyou for answering! May I ask where the launcher settings are??? Which launcher are we even talking about?

**Comment by Spirit117** [comment]
I figured it out, I'm dumb (this is what I get for trying something at 2am).

The mods folder needed to go in file path e:steam/steampps/common/xcom2/Xcom2-war of the chosen/Xcom game/mods.

I didn't realize they needed to be in Xcom game. Ofc, when I clicked on Xcom game and there were already a mods folder in there I was like ohhhhh.... I'm stupid. 

From there, as soon as you launch Xcom 2, it will detect the mods as launch able.
